docker-compose.yml

```diff
@@ -167,6 +167,22 @@ services:
         - "traefik.http.services.keycloak.loadbalancer.server.port=8080"
         - "traefik.http.routers.keycloak.priority=10"
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.3.3
+    networks:
+      - clowder2
+    restart: unless-stopped
+    environment:
+      - "cluster.name=clowder2"
+      - "discovery.type=single-node"
+      - "xpack.security.enabled=false"
+      - "xpack.security.http.ssl.enabled=false"
+    volumes:
+      - elasticsearch:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
```

Comment:
In the case of the production docker container the ports should not be open to the world. The webapp should be able to connect to the container on the internal docker network. Could you remove them and test the connection?

Reply:
I tried this, I am not able to connect to elasticsearch server. 

Also, what's the difference between docker-compose.yml & docker-compose.dev.yml? The reason I asked because I see ports mentioned for other services in docker-compose.dev.yml.